### PR TITLE
Added support for Visual Studio 2010 Ultimate

### PR DESCRIPTION
**Pull request by abhaymittal**

### CODE REVIEW

File: attributes/vs2010.rb [hunk 1]
```diff
@@ -33,6 +33,16 @@
...
 
+# VS 2010 Ultimate
+default['visualstudio']['2010']['ultimate']['filename'] =
+  'en_visual_studio_2010_ultimate_x86_dvd_509116.iso'
+default['visualstudio']['2010']['ultimate']['package_name'] =
+  'Microsoft Visual Studio 2010 Ultimate - ENU'
+default['visualstudio']['2010']['ultimate']['checksum'] =
+  '602926C864FA1DDC087FEFB1ED952216DA4122D4DB774050D8E9373D3E4BFF98'
+default['visualstudio']['2010']['ultimate']['installer_file'] = File.join('setup', 'setup.exe')
+default['visualstudio']['2010']['ultimate']['config_file'] = nil
```
Comment (abhaymittal) [inline attributes/vs2010.rb]: Okay. The recipe was already using single quotes so I stuck to that convention only.

**Comment by heathsnow** [comment]
I've removed Hound integration for this cookbook.  It should hound us no more.

**Comment by sneal** [comment]
Thanks!

**Comment by abhaymittal** [comment]
Thank you for accepting the edit :)

**Pull request merged**